### PR TITLE
Pinned rspec_puppet to 2.3.2

### PR DESCRIPTION
**Pull request by jonnytpuppet**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,7 +18,12 @@ group :development, :unit_tests do
   gem 'puppet_facts',              :require => false
   gem 'puppet-blacksmith',         :require => false
   gem 'puppetlabs_spec_helper',    :require => false
-  gem 'rspec-puppet', '>= 2.3.2',  :require => false
+
+  # Modulesync will override the following. This is only a temporary measure until we fix
+  # our tests for rspec-puppet 2.4.0 support.
+  # Actual update work to be done in https://tickets.puppetlabs.com/browse/FM-5035
```
Comment (daenney) [inline Gemfile]: Can't actually view that, don't have the perms.

**Comment by daenney** [comment]
What's going on? The ticket linked in the comments isn't viewable by community folks.

**Comment by tphoney** [comment]
@daenney @jonnytpuppet i moved the ticket https://tickets.puppetlabs.com/browse/MODULES-3238 so it is visable

**Comment by DavidS** [comment]
Superseded by #599

**Pull request closed without merging**